source/vocab/details.ttl

```diff
@@ -1493,10 +1494,14 @@
     rdfs:subPropertyOf sdo:spatialCoverage ;
     owl:equivalentProperty bf2:geographicCoverage .
 
-:temporalCoverage a owl:DatatypeProperty;
+:TemporalCoverage a owl:Class ;
+    rdfs:subClassOf :TemporalQualification ;
+    rdfs:label "temporal coverage"@en, "temporal täckning"@sv .
+
+:temporalCoverage a owl:ObjectProperty ;
     rdfs:label "Temporal coverage"@en, "Kronologisk täckning"@sv;
-    rdfs:domain :Work;
-    rdfs:range rdfs:Literal;
+    rdfs:domain :Work ;
+    rdfs:range :TemporalCoverage ;
     rdfs:comment "Kronologisk täckning avseende innehållet i resursen."@sv;
```

Comment:
"kronologisk" vs "temporal" täckning

Reply:
Addressed with "rationale" in c3e2e46645dcd75ebe7c4f044479bcb91719b8d4